src/main/java/edu/harvard/iq/dataverse/engine/command/impl/DeletePidCommand.java

```diff
@@ -47,7 +47,7 @@ protected void executeImpl(CommandContext ctxt) throws CommandException {
         GlobalIdServiceBean idServiceBean = GlobalIdServiceBean.getBean(protocol, ctxt);
         try {
             // idServiceBean.deleteIdentifier(dataset); // didn't work
-            String baseUrl = System.getProperty("doi.baseurlstringnext");
+            String baseUrl = ctxt.systemConfig().getDataCiteRestApiUrlString();
```

Comment:
Again, surprised that this command only supports DataCite, but it was like this previously.